### ISSUE TITLE
New rule: Visible label is part of accessible name

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#482]
The ACT-Rules community group has completed work on {{rule name}}. We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Visible label is part of accessible name
**Rule id**: 2ee8b8
**Link to the rule**: https://act-rules.github.io/rules/2ee8b8
**License**: W3C Software And Document Notice And License
**Implementors**:
- axe-core
- Qualweb
- SortSite

If you'd like to contact me, please e-mail me at wilco.fiers@deque.com.com. You can also reach me on the W3C Community Slack, my name there is Wilco Fiers.

**Comment by maryjom** [comment]
[Survey results](https://www.w3.org/2002/09/wbs/93339/ACTVisibleLabel/results) for the survey due on 24 September.

**Comment by maryjom** [comment]
[Issue 491](https://github.com/w3c/wcag-act/issues/491) opened from survey results. Discussed in the [24 Sept. meeting](https://www.w3.org/2020/09/24-wcag-act-minutes.html#item02) and [17 Sept. meeting](https://www.w3.org/2020/09/17-wcag-act-minutes.html#item03).
